config/locales/docs/sk.yml

```diff
@@ -866,7 +866,7 @@ sk:
             <<: *effective_range_columns
             <<: *timestamp_columns
         organization_registration_office_entries:
-          description: Registrátor právnických osôb
+          description: Registrátory právnických osôb
```

Comment:
asi skor `Registratori`
(tiez som si nebol uplne isty, ale [pozri tu](https://www.kvizy.eu/slovensky-jazyk/tvary-slova/registr%C3%A1tor?word_id=41758))
 
inak isto je to spravne? "registrator" pozeram ze google vyhadzuje len registratorov domen. nebude skor presnejsi preklad "registracny urad" alebo "registrovy sud"?

Reply:
asi som nasiel - inak dobry source je toto:  [RPO search](https://rpo.statistics.sk/rpo/#search)

![image](https://user-images.githubusercontent.com/38187837/110126201-7bdbda80-7dc4-11eb-90af-acf683f83368.png)

Reply:
Áno, chcela som aj ja najprv tak dať, ale už v existujúcich popisoch iných tabuliek som našla preklad pre _registration_office_ ako "Registrátor", tak som dala podľa toho.
Ako teda?

Reply:
nasiel som `other_registrar_name` ako `Registrator` - tu prosim pozri kontext (ci to "registrar" je nas umelo nazvany stlpec, alebo to je opisane zo zdrojovych dat)
lebo k tomu su asi dve moznosti - prva (pravdepodobne) ze to sa u nas (nespravne) urcilo ako "registrator", v tom pripade to tiez normalne zmenme na "registrovy urad" (teda samozrejme, ak ide o tu istu vec)
- druha moznost, ze to je aj v zdroji ako registrar (ppr. registrator) ...  aj tak to asi dajme ako registrovy urad, resp. prelozme to spravne